### PR TITLE
Fixes #49 - Don't export/collect resources if storeconfigs is not ena…

### LINKED ISSUE #49: storeconfigs
Hi, 

a puppet parser validate manifests/init.pp yields:
Warning: You cannot collect without storeconfigs being set on line 128 in file /etc/puppet/environments/production/modules/hosts/manifests/init.pp
Warning: You cannot collect exported resources without storeconfigs being set; the collection will be ignored on line 134 in file /etc/puppet/environments/production/modules/hosts/manifests/init.pp
Warning: You cannot collect exported resources without storeconfigs being set; the collection will be ignored on line 138 in file /etc/puppet/environments/production/modules/hosts/manifests/init.pp

What's going on?

Regards,

Rudi G.

### PR DESCRIPTION
**Pull request by edestecd**
…bled

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -120,7 +120,7 @@
     ip           => $localhost6_ip,
   }
 
-  if $use_fqdn_real == true {
+  if $::settings::storeconfigs and $use_fqdn_real == true {
```
Comment (ghoneycutt) [inline manifests/init.pp]: If use_fqdn is true and we are not using storeconfigs, there should still be the `host { $::fqdn` entry, though without the `@@`

**Comment by ghoneycutt** [comment]
Closing in favor of PR #42 

Thank you!

File: manifests/init.pp (hunk 1 above)
Comment (edestecd) [inline manifests/init.pp]: I agree

**Comment by ghoneycutt** [comment]
Thanks @edestecd 

I'm testing the other PR now. Hope to have something going soon.

**Pull request closed without merging**